rules/linux/defense_evasion_attempt_to_disable_iptables_or_firewall.toml

```
@@ -17,10 +17,40 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Attempt to Disable IPTables or Firewall"
+note = """## Triage and analysis
+
+### Investigating Attempt to Disable IPTables or Firewall
+
+Iptables is a firewall program for Linux. It will monitor traffic from and to your server using tables.
+These tables contain sets of rules, called chains, that will filter incoming and outgoing data packets.
+The rule identifies executions to disable iptables or firewalld in an attempt to affect how a host is allowed to
+receive or send network traffic.
+
+#### Possible investigation steps
+
+- Identify the session entry leader and session user.
+- Examine the contents of session leading to the iptables or firewalld disable exectuions via the Session View.
```

Added a link here so users can refer to our Session View docs if they're new to using the tool. 

@benironside since you write the Session View docs and are more familiar with Linux, would you mind double-checking the language here? I swapped "executions" with "commands" and changed "disable" to "turn off".

```suggestion
- Use [Session View](https://www.elastic.co/guide/en/security/master/session-view.html) to examine the session and locate commands that turned off iptables or firewalld.
```